src/manager.wsgi

```diff
@@ -373,16 +381,17 @@ def application(environ, start_response):
                                   "<tr><td colspan=\"2\">%s <code>retrace-server-interact %s shell</code></td></tr>" \
                                   "<tr><td colspan=\"2\">%s <code>retrace-server-interact %s %s</code></td></tr>" \
                                   "<tr><td colspan=\"2\">%s <code>man retrace-server-interact</code> %s</td></tr>" \
-                                  % (_("This is an interactive task"), _("You can jump to the chrooted shell with:"), filename,
-                                     _("You can jump directly to the debugger with:"), filename, debugger,
-                                     _("see"), _("for further information about cmdline flags"))
+                                  % (_("This is an interactive task"), _("You can jump to the chrooted shell with:"), \
```

Comment:
The `\` at the end of the lines are not needed because everything is in brackets. The indentation should be left unchanged.